### ISSUE TITLE
test pre-trained model on my own dataset

### ISSUE THREAD

**Issue by minyeo** [likyoo/open-cd#81]
when I test the pre-trained S2Looking model on my own dataset, it doesn't working. 
Is it not possible to apply pretrained models to different datasets? Or did I do something wrong? 
The object in the middle box has moved.

![image](https://github.com/likyoo/open-cd/assets/58988225/0e62050d-6677-4f02-85f5-5bccee4d0f4f)
![image](https://github.com/likyoo/open-cd/assets/58988225/7dff9799-8d62-4fa8-8920-a05d25a4f996)
![image](https://github.com/likyoo/open-cd/assets/58988225/f8d20a1c-6fce-4dfa-a26e-5e40b521deed)

**Comment by mustafacavusoglu** [comment]
yes, you have to train your model using your dataset to get good result.
